src/transport/http/v03/headers.ts

```diff
@@ -0,0 +1,35 @@
+import { PassThroughParser, MappedParser } from "../../../parsers";
+
+const TYPE = "ce-type";
+const SPEC_VERSION = "ce-specversion";
+const SOURCE = "ce-source";
+const ID = "ce-id";
+const TIME = "ce-time";
+const SUBJECT = "ce-subject";
+const HEADER_CONTENT_TYPE = "content-type";
+const CONTENT_ENCODING = "ce-datacontentencoding";
+const SCHEMA_URL = "ce-schemaurl";
+
+export const BINARY_HEADERS = Object.freeze({
+  TYPE, SPEC_VERSION, SOURCE, ID, TIME, CONTENT_ENCODING, SUBJECT, SCHEMA_URL
+});
+
+const passThrough = new PassThroughParser();
+function parser(header: string, parser = passThrough): MappedParser {
+  return { name: header, parser };
+}
+
+/**
+ * A utility Map used to retrieve the header names for a CloudEvent
+ * using the CloudEvent getter function.
+ */
+export const headerMap: Map<string, MappedParser> = new Map();
+headerMap.set("dataContentType", parser(HEADER_CONTENT_TYPE));
+headerMap.set("dataContentEncoding", parser(CONTENT_ENCODING));
+headerMap.set("subject", parser(SUBJECT));
+headerMap.set("type", parser(TYPE));
+headerMap.set("specversion", parser(SPEC_VERSION));
+headerMap.set("source", parser(SOURCE));
+headerMap.set("id", parser(ID));
+headerMap.set("time", parser(TIME));
+headerMap.set("schemaURL", parser(SCHEMA_URL));
```

Comment:
Let's use a raw JavaScript object instead of a Map. If there is a very strong reason for using a Map, follow the below suggestion.

---

Let's use a `Map` constructor with key value pairs rather than using the `set` method N times.

Example:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/Map